pkg/lib/operatorclient/customresources.go

```diff
@@ -154,6 +154,49 @@ func (c *Client) UpdateCustomResourceRaw(apiGroup, version, namespace, resourceP
 	return nil
 }
 
+// UpdateCustomResourceStatus updates the custom resource's status.
+func (c *Client) UpdateCustomResourceStatus(item *unstructured.Unstructured) error {
```

Comment:
The operatorclient is more or less deprecated in favor of direct calls via the generated clients or the unstructured client. Can we just remove this? (less to test, too)

Reply:
Oh this code is not in use anymore and I should discard this. I fully use the OC client to create/update CR.